desktop/cmp/persistenceManager/impl/ManageDialog.ts

```diff
@@ -0,0 +1,160 @@
+import {form} from '@xh/hoist/cmp/form';
+import {grid} from '@xh/hoist/cmp/grid';
+import {br, div, filler, fragment, hframe, placeholder, spacer, vframe} from '@xh/hoist/cmp/layout';
+import {hoistCmp, uses, XH} from '@xh/hoist/core';
+import {button} from '@xh/hoist/desktop/cmp/button';
+import {formField} from '@xh/hoist/desktop/cmp/form';
+import {switchInput, textArea, textInput} from '@xh/hoist/desktop/cmp/input';
+import {mask} from '@xh/hoist/desktop/cmp/mask';
+import {panel} from '@xh/hoist/desktop/cmp/panel';
+import {toolbar} from '@xh/hoist/desktop/cmp/toolbar';
+import {fmtCompactDate} from '@xh/hoist/format';
+import {Icon} from '@xh/hoist/icon';
+import {dialog} from '@xh/hoist/kit/blueprint';
+import {ManageDialogModel} from './ManageDialogModel';
+
+export const manageDialog = hoistCmp.factory<ManageDialogModel>({
+    displayName: 'ManageDialog',
+    model: uses(ManageDialogModel),
+
+    render({model}) {
+        return dialog({
+            isOpen: true,
+            icon: Icon.gear(),
+            title: `Manage ${model.parentModel.capitalPluralNoun}`,
+            className: 'persistence-manager__manage-dialog',
+            style: {width: 800, height: 475},
```

Comment:
This seems like it might be a wide width for the framework to assume will be available.  Might be worth setting a `maxWidth` of `90vw` to accommodate for smaller browser windows